src/wasm.h

```diff
@@ -1344,6 +1349,8 @@ class Module {
   Name start;
 
   std::vector<UserSection> userSections;
+
+  // Source maps debug info.
```

Comment:
Maybe a separate change but we should probably rename these fields to make it more clear.

Reply:
I hope to remove the source maps code in the not far future, which would remove any confusion between these fields. (Once dwarf line numbers works, we can use that for source maps.)